lib/modules/versioning/npm/range.ts

```diff
@@ -63,6 +64,9 @@ export function getNewValue({
   currentVersion,
   newVersion,
 }: NewValueConfig): string | null {
+  if (rangeStrategy === 'bump' && isSemVerXRange(currentValue)) {
```

Comment:
this should have a test in npm versioning folder too, so testing this folder only give full coverage

Reply:
This is already covered 
` yarn jest  --testPathPattern=lib/modules/versioning/npm`

![image](https://user-images.githubusercontent.com/97394622/195806748-6579ff60-729f-460d-adac-b6c927c78105.png)

but this folder is not fully self covered it seems, regardless of this PR.

![image](https://user-images.githubusercontent.com/97394622/195806601-b31bde70-46cb-4910-82f2-c384c7f23cad.png)

I would gladly cover do it in a separate PR

Reply:
ok